cmd/cli/serve/util.go

```diff
@@ -143,84 +177,43 @@ func GetRequesterConfig(ctx context.Context, cfg types.BacalhauConfig, createJob
 	return requesterConfig, nil
 }
 
-func getNodeType(cfg types.BacalhauConfig) (requester, compute bool, err error) {
-	requester = false
-	compute = false
-	err = nil
-
-	for _, nodeType := range cfg.Node.Type {
-		if nodeType == "compute" {
-			compute = true
-		} else if nodeType == "requester" {
-			requester = true
-		} else {
-			err = fmt.Errorf("invalid node type %s. Only compute and requester values are supported", nodeType)
-		}
-	}
-	return
-}
-
-func getNetworkConfig(networkStorePath string, cfg types.NetworkConfig) (node.NetworkConfig, error) {
-	return node.NetworkConfig{
-		Port:                     cfg.Port,
-		AdvertisedAddress:        cfg.AdvertisedAddress,
-		Orchestrators:            cfg.Orchestrators,
-		StoreDir:                 networkStorePath,
-		AuthSecret:               cfg.AuthSecret,
-		ClusterName:              cfg.Cluster.Name,
-		ClusterPort:              cfg.Cluster.Port,
-		ClusterAdvertisedAddress: cfg.Cluster.AdvertisedAddress,
-		ClusterPeers:             cfg.Cluster.Peers,
-	}, nil
-}
-
-func getExecutionStore(ctx context.Context, storeCfg types.JobStoreConfig, path string) (store.ExecutionStore, error) {
-	if err := storeCfg.Validate(); err != nil {
-		return nil, err
-	}
-
-	switch storeCfg.Type {
-	case types.BoltDB:
-		return boltdb.NewStore(ctx, path)
-	default:
-		return nil, fmt.Errorf("unknown JobStore type: %s", storeCfg.Type)
-	}
-}
-
-func getJobStore(ctx context.Context, storeCfg types.JobStoreConfig, path string) (jobstore.Store, error) {
-	if err := storeCfg.Validate(); err != nil {
-		return nil, err
+func getNetworkConfig(cfg types2.Bacalhau) (node.NetworkConfig, error) {
+	// TODO(review) see method getPublicNATSOrchestratorURL for why we ignore the host.
+	_, listenPortStr, err := net.SplitHostPort(cfg.Orchestrator.Listen)
```

Comment:
to remain consistent with our current behavior as defined in https://github.com/bacalhau-project/bacalhau/blob/a3c2b92f75da951359161ea6f736799d2133d994/cmd/cli/serve/serve.go#L312-L323 I have omitted setting a host field on the `NetworkConfig`, also it doesn't have a host field.